.github/workflows/check-standard.yaml

```diff
@@ -0,0 +1,79 @@
+on:
```

Comment:
It looks like another option here could be to run this on every push, which would more closely mirror the current workflow with Travis CI. 

```
on: [push, pull_request]

name: R-CMD-check
```

Taken from: https://github.com/r-lib/rlang/blob/master/.github/workflows/R-CMD-check.yaml